app/src/main/java/com/mediaan/mymediaan/model/Interests.kt

```diff
@@ -0,0 +1,28 @@
+package com.mediaan.mymediaan.model
+
+enum class Interest {
+    MUSIC,
+    SPORTS,
+    HIKING,
+    DOGS,
+    CATS,
+    READING,
+    TRAVELING,
+    COOKING,
+    GAMING,
```

Comment:
This could be split into two values - gaming and boardgames

Reply:
...I may or may not have asked ChatGPT to generate this list. Updating it.